econml/cate_estimator.py

```diff
@@ -5,8 +5,10 @@
 
 import abc
 import numpy as np
+import pandas as pd
 from functools import wraps
 from copy import deepcopy
+from sklearn.utils import check_array
```

Comment:
The new imports here shouldn't be needed if there are no other changes to this file, right?
```suggestion
from functools import wraps
from copy import deepcopy
```